BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Group/GroupListModalView.js

```diff
@@ -0,0 +1,102 @@
+import OrchestraView        from '../OrchestraView'
+import ModalView            from '../../../Service/Modal/View/ModalView'
+import UserGroups           from '../../Collection/Group/UserGroups'
+import GroupListForUserView from './GroupListForUserView'
+import Application          from '../../Application'
+
+/**
+ * @class GroupListModalView
+ */
+class GroupListModalView extends ModalView
+{
+    /**
+     * @inheritdoc
+     */
+    preinitialize() {
+        this.tagName = 'div';
+        this.className = 'modal fade';
+        this.events = {
+            'hidden.bs.modal': 'hide',
+            'click .select-group': '_selectGroup',
+            'click .search-engine button.submit': '_search',
+            'click .search-engine button.reset': '_search'
+        }
+    }
+
+    /**
+     * Initialize
+     * @param {Sites}    sites
+     * @param {array}    blockedGroups
+     * @param {array}    selectedGroups
+     */
+    initialize({sites, blockedGroups, selectedGroups}) {
+        this._sites = sites;
+        this._blockedGroups = blockedGroups;
+        this._selectedGroups = selectedGroups;
+    }
+
+    /**
+     * Render Site selector
+     */
+    render() {
+        this._collection = new UserGroups();
+        this._groupListForUserView = new GroupListForUserView({
+            collection: this._collection,
+            blockedGroups: this._blockedGroups,
+            selectedGroups: this._selectedGroups
+        });
+
+        let template = this._renderTemplate('User/groupListModalView',
+            {
+                sites: this._sites.models,
+                language: Application.getContext().language
+            }
+        );
+        this.$el.html(template);
+        $('.modal-body .groups-list', this.$el).html(this._groupListForUserView.render().$el);
+
+        return this;
+    }
+
+    /**
+     * Search node in list
+     * @param {Object} event
+     *
+     * @returns {boolean}
+     * @private
+     */
+    _search(event) {
+        event.stopPropagation();
+        let filters = {};
+        if ($(event.target).hasClass('submit')) {
+            $('button.reset', this.$el).removeClass('hidden');
+        }
+        if ($(event.target).hasClass('reset')) {
+            $('button.reset', this.$el).addClass('hidden');
+            $('form.search-engine', this.$el).trigger('reset');
+        }
+
+        let formData = $('form.search-engine', this.$el).serializeArray();
+        for (let data of formData) {
+            filters[data.name] = data.value;
+        }
+
+        this._groupListForUserView.filter(filters);
+
+        return false;
+    }
+
+    _selectGroup() {
+        let formGroups = _.pluck($('[name="group"]', this.$el).removeAttr('disabled').serializeArray(), 'value');
+        let selectedGroups = [];
+        for (let group of this._collection.models) {
```

Comment:
I do something like this : 
'
        let formGroups = _.pluck($('[name="group"]', this.$el).removeAttr('disabled').serializeArray(), 'value');
        formGroups = _.map(formGroups, function(item, id) {
            return {
                "id " : item
            };
        });
        let selectedGroups = this._collection.where(formGroups);
'
But not working...